src/main/scripts/testicat

```diff
@@ -10,21 +10,21 @@ def check(r):
         if r.status_code / 100 != 2:
             json = r.json()
             if json.get("offset"):
-                print >> sys.stderr, json["code"], json["message"], json["offset"]
+                print(json["code"], json["message"], json["offset"], file=sys.stderr)
```

Comment:
This seems incompatable with python 2.7

Reply:
I guess it will work if you add a
```python
from __future__ import print_function
```
statement as very first line of the script.

Reply:
@l-mas @stuartpullinger has this been resolved?